### DISCUSSION TITLE
Private Pypi python package that's non "main" version decision for Renovatebot's PR (always keeps recommending main)

### DISCUSSION THREAD

**Discussion by itzikban** [renovatebot/renovate#19770] [Request Help]
Let's say we have a package named pythonGlobals.
The latest master release was 2.0.1 which brings us to pythonGlobals==2.0.1

We have a feature branch named dev in which the latest package created was pythonGlobals==2.0.1b0+dev.**5** (PEP440 standard)
In our repository which has pythonGlobals in its requirements.txt file we have pythonGlobals==2.0.1b0+dev.**1** as a depdenency.

What we expect:
Renovate should offer me pythonGlobals==2.0.1b0+dev.**5** in the PR

What actually happens:
Renovate offers me the latest master release: pythonGlobals==2.0.1, which is older than my latest dev package.
Thanks a lot for your help !

logs : 

                 "currentValue": "==2.0.1b0+dev.1",
                 "datasource": "pypi",
                 "currentVersion": "2.0.1b0+dev.1",
                 "depIndex": 5,
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "2.0.1",
                     "newValue": "==2.0.1",
                     "newMajor": 2,
                     "newMinor": 0,
                     "updateType": "patch",
                     "isRange": true,
                     "branchName": "renovate/pythonglobals-2.x"
                   }
                 ],
                 "warnings": [],
                 "versioning": "pep440",
                 "isSingleVersion": true,
                 "fixedVersion": "2.0.1b0+dev.1"
               }
             ],

**Comment by viceice** [comment]
expected, because the full release is higher than the beta release. you can play with allowed version and ignore unstable config options in a package rule for that dependency

**Reply by itzikban** [reply]
Hi Thanks for your help I tried but didnt work, does my versioning topology is it correct ?
master = 2.0.1

new py tag for the new branch
dev = 2.0.1b0+dev.1
....
the third commit on branch > 2.0.1b0+dev.3 = new tag suggestion should be.
this is how its should be?
this is the logs after unstable configured:

Thanks again !

DEBUG: Env config
"config": {
"hostRules": [],
"baseDir": "/tmp/renovate/",
"platform": "github",
"token": "***********",
"autodiscover": false,
"baseBranches": ["conan"],
"ignoreUnstable": false
}

           {
             "depName": "pythonGlobals",
             "currentValue": "==2.0.1b0+dev.1",
             "datasource": "pypi",
             "currentVersion": "2.0.1b0+dev.1",
             "depIndex": 5,
             "updates": [
               {
                 "bucket": "non-major",
                 "newVersion": "2.0.1",
                 "newValue": "==2.0.1",
                 "newMajor": 2,
                 "newMinor": 0,
                 "updateType": "patch",
                 "isRange": true,
